spec/services/users/profile_spec.rb

```diff
@@ -429,6 +429,7 @@
     end
 
     describe '#vet360_contact_information' do
+      Flipper.disable(:remove_pciu)
```

Comment:
Is there a reason Flipper is directly disabled instead of being stubbed throughout specs? It seems to me this is likely to cause flaky specs

Reply:
@bosawt These modifications were corrected in another PR. I removed the remove_pciu references